tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -57,6 +59,13 @@ ifelse(PLATFORM, `adl',
 `	define(`SMART_ALH_INDEX', 0x202)
 	define(`SMART_ALH_PLAYBACK_NAME', `SDW2-Playback')
 	define(`SMART_ALH_CAPTURE_NAME', `SDW2-Capture')
+	# Add BT audio offload support
+	define(`BT_PIPELINE_PB_ID', `14') dnl DMIC_PIPELINE_KWD_ID + 1
+	define(`BT_PIPELINE_CP_ID', `15') dnl DMIC_PIPELINE_KWD_ID + 2
+	define(`BT_DAI_LINK_ID', `10')
+	define(`BT_PCM_ID', `9') dnl added after HDMI
+	define(`HW_CONFIG_ID', `10')
```

Comment:
same, what is this?

Reply:
HW_CONFIG_ID was introduced for MULTI_SSP_CONFIG:
https://github.com/thesofproject/sof/blob/main/tools/topology/platform/common/ssp.m4#L55

Which is not needed for SSP_CONFIG
https://github.com/thesofproject/sof/blob/main/tools/topology/platform/common/ssp.m4#L20

Reply:
oh man, that's a terrible macro. There should some sort of prefix of something that says what scope this applies to. @ranj063 @kv2019i @juimonen can we fix this?